tic_tac_toe/game.py

```diff
@@ -24,17 +26,28 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if not isinstance(position, tuple):
+        return False
+    if len(position) != 2:
+        return False
+    if position[0] not in [0,1,2]:
+        return False
+    if position[1] not in [0,1,2]:
+        return False
+    return True
 
 def _board_is_full(board):
     """
     Returns True if all positions in given board are occupied.
 
     :param board: Game board.
     """
-    pass
-
+    for x in range(2):
```

Comment:
Range goes up to but not including the limit. So you'd want a `range(3)` there instead of a `range(2)`

ie, `range(2)` would only give you 0 and 1